adafruit_fancyled/adafruit_fancyled.py

```diff
@@ -13,6 +13,7 @@
 
 * Author(s): PaintYourDragon
 """
+from __future__ import annotations
```

Comment:
I was incorrect about this. That import can and should be at the beginning of the file rather than inside the try block. Sorry for pointing you wrong on this. I've pushed a commit that moves this back to the top.